### DISCUSSION TITLE
Not found test VisLoc details in supplementary material

### DISCUSSION THREAD

**Discussion by FuRong0912** [zju3dv/LoFTR#20] [General]
Not found the implementation details of testing LoFTR on VisLoc in the supplementary material.

**Comment by zehongs** [comment]
Hi, sorry for the late reply
1. For aachen-local-feature-evaluation, we followed the pipeline from [dualrc-net](https://github.com/ActiveVisionLab/DualRC-Net/tree/master/aachen_day_and_night) and [local-feature-evaluation](https://github.com/tsattler/visuallocalizationbenchmark/tree/master/local_feature_evaluation).
2. For aachen and inloc, we used the code and image retrieval results from [hloc-toolbox](https://github.com/cvg/Hierarchical-Localization).

**Reply by HUSTNO1WXY** [reply]
I don't find the implementation code of testing LoFTR on HPatches, can you show me a reference link? Thank you
